package.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "frontend-coding-standards",
   "version": "1.0.0",
+  "private": false,
   "description": "MediaMonks - Frontend Coding Standards",
   "repository": "git@github.com:mediamonks/frontend-coding-standards.git",
-  "author": "Thijs <thijs@mediamonks.com>",
   "license": "MIT",
-  "private": false,
+  "author": "Thijs <thijs@mediamonks.com>",
```

Comment:
Should we have a generic email for inquiries ? So the author can be something like Media.Monks Frontend <frontend@mediamonks.com>.

Reply:
please create a separate issue for this